src/sidebar/services/api-routes.js

```diff
@@ -19,11 +19,11 @@ function apiRoutes($http, settings) {
         'Hypothesis-Client-Version': '__VERSION__', // replaced by versionify
       },
     };
-    return $http.get(url, config).then(({ status, data }) => {
-      if (status !== 200) {
+    return fetch(url, config).then(response => {
+      if (response.status !== 200) {
```

Comment:
Remind me which status codes will cause a `fetch` Promise to reject again?

Reply:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) fetch never rejects based on a status code:

> A fetch() promise only rejects when a network error is encountered (which is usually when there’s a permissions issue or similar). A fetch() promise does not reject on HTTP errors (404, etc.). Instead, a then handler must check the Response.ok and/or Response.status properties.